dnsapi/dns_octodns.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env sh
+
+# This API is a wrapper for other API so that you
+# are able to put your TXT record to multiple providers.
+# The functionality is in the responsibility of the respective API
+# therefore please check if your used APIs work.
+# e.g. for using NS1 and AWS Route53, use:
+# OCTODNS_PROVIDERS=nsone_aws
+# --dns octodns
+#
+# Author: Josef Vogt
+#
+########  Public functions #####################
+dns_octodns_add() {
+  fulldomain=$1
+  txtvalue=$2
+  _info "Creating DNS entries via octoDNS"
+  _debug fulldomain "$fulldomain"
+  _debug txtvalue "$txtvalue"
+  if [ -z "$OCTODNS_PROVIDERS" ]; then
+    OCTODNS_PROVIDERS=""
+    _err "You didn't specify OCTODNS_PROVIDERS yet."
+    _err "Please specifiy your providers and try again."
+    return 1
+  fi
+
+  #save the providers list to the account conf file.
+  _saveaccountconf OCTODNS_PROVIDERS "$OCTODNS_PROVIDERS"
```

Comment:
don't use `_saveaccountconf`

see the example about what and how we use:
https://github.com/acmesh-official/acme.sh/blob/master/dnsapi/dns_cf.sh

Reply:
Right; I took dns_nsone.sh as an example. This API uses `saveaccountconf` unfortunately.
I change it to `_saveaccountconf_mutable`.